params/config.go

```diff
@@ -137,8 +137,10 @@ var (
 	}
 
 	DefaultChainConfig = &ChainConfig{big.NewInt(1337), big.NewInt(0), nil, false, big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), nil, &IstanbulConfig{
```

Comment:
I think the defauls should be `mainnet` or non existent.

If not, we risk using a dev config on a mainnet node if we missplace a config.

Reply:
Checked that mainnet is default, and changed the name of this config to `IstanbulTestConfig`.